packages/mui-codemod/src/v5.0.0/base-remove-unstyled-suffix.test/actual.js

```diff
@@ -0,0 +1,9 @@
+import ButtonUnstyled, {
```

Comment:
```suggestion
import ButtonUnstyled, {
```

we should have an option about whether we should rename the default imports. We can rename them by default, but I feel like some people may want to use still `ButtonUnstyled` mostly because of conflicts with other imports.

Reply:
Agree. I wrote another codemod script that doesn't remove unstyled suffix from default imports. https://github.com/mui/material-ui/pull/36873/commits/4bcdd3e9e39bfb77b07894245a0177106b3a071d

Reply:
Update: We agree that the codemod should not change any of the importees names, as that may break the code of the developers.